src/components/Footer/Footer.js

```diff
@@ -199,7 +200,7 @@ const Footer = ({
           )}
           {disableComment ? (
             <View testID="disableComment" style={styles.btn}>
-              <View style={styles.btnComment}>
+              <View style={styles.btnComment(0)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `styles.btnComment` is now being called with an argument of 0. Make sure this change doesn't affect the style or functionality of the component.
```diff
- <View style={styles.btnComment}>
+ <View style={styles.btnComment(0)}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->